analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.CSharp8.cs

```diff
@@ -50,14 +59,14 @@ void LocalFunction()
         {
             var d = new Disposable();
             d.Dispose();
-            d.Dispose(); // FIXME Non-compliant - FN: local functions are not supported by the CFG
+            d.Dispose(); // FN: local functions are not supported
         }
 
         static void LocalStaticFunction()
         {
             var d = new Disposable();
             d.Dispose();
-            d.Dispose(); // FIXME Non-compliant - FN: local functions are not supported by the CFG
+            d.Dispose(); // FN: local functions are not supported
```

Comment:
My understanding is that the SE does not go into (from what I saw in the SymbolicExectutionRunner).
Could you please confirm?